### PR TITLE
[CBRD-23816] modified answer for support HASH TEMP FILE

### PR DESCRIPTION
**Pull request by tw-kang**
http://jira.cubrid.org/browse/CBRD-23816

### CODE REVIEW

File: sql/_13_issues/_20_2h/answers/cbrd_23665.answer_cci [hunk 1]
```diff
@@ -949,7 +949,7 @@ trace
 Trace Statistics:
   SELECT (time: ?, fetch: ?, ioread: ?)
     SCAN (temp time: ?, fetch: ?, ioread: ?, readrows: ?, rows: ?)
-      SCAN (hash temp(h), build time: ?, time: ?, fetch: ?, ioread: ?, readrows: ?, rows: ?)
+      SCAN (hash temp(f), build time: ?, time: ?, fetch: ?, ioread: ?, readrows: ?, rows: ?)
```
Comment (shparkcubrid) [inline sql/_13_issues/_20_2h/answers/cbrd_23665.answer_cci]: It is necessary to check whether the answer sheet should be changed or the test case should be changed.

File: sql/_13_issues/_20_2h/answers/cbrd_23665.answer_cci [hunk 2]
```diff
@@ -913,7 +913,7 @@ trace
 Trace Statistics:
   SELECT (time: ?, fetch: ?, ioread: ?)
     SCAN (temp time: ?, fetch: ?, ioread: ?, readrows: ?, rows: ?)
-      SCAN (hash temp(h), build time: ?, time: ?, fetch: ?, ioread: ?, readrows: ?, rows: ?)
+      SCAN (hash temp(m), build time: ?, time: ?, fetch: ?, ioread: ?, readrows: ?, rows: ?)
```
Comment (shparkcubrid) [inline sql/_13_issues/_20_2h/answers/cbrd_23665.answer_cci]: It is necessary to check whether the answer sheet should be changed or the test case should be changed.

Comment (tw-kang) [reply sql/_13_issues/_20_2h/answers/cbrd_23665.answer_cci]: in the testcase seems to check memory limit.
case line is 
select /*+ ordered */ count(*) from (select * from t1 limit 1) a, (select * from t1 limit 100000) b where a.a = b.a and a.b = b.b and a.c = b.c;

does this statement affect memory limit?

File: sql/_13_issues/_20_2h/answers/cbrd_23665.answer_cci (hunk 1 above)
Comment (tw-kang) [reply sql/_13_issues/_20_2h/answers/cbrd_23665.answer_cci]: before was setting system parameter 'max_hash_list_scan_size=4M'.
now default increased to 8M.
does that mean size minimum value starts from 8M? if yes test case have to be modified.

**Review by ssihil — APPROVED** [review]

**Pull request merged**